openlibrary/plugins/admin/code.py

```diff
@@ -15,10 +15,10 @@
 from infogami.utils import delegate
 from infogami.utils.view import render, public
 from infogami.utils.context import context
-from infogami.utils.view import add_flash_message
+from infogami.utils.flash import add_flash_message
```

Comment:
The incompatible change to infogami has been reverted, so these imports can be put back the way they were.

Reply:
I just pushed a commit that reverts the 7 commits that made these changes, so hopefully this is all set.